src/Components/Layout/NavBar.js

```diff
@@ -6,6 +6,7 @@ import {
   faCreativeCommonsBy,
 } from "@fortawesome/free-brands-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Button from 'react-bootstrap/Button'
```

Comment:
Adhere to the new import guidelines. Refer to #33

Reply:
Done for this file as well as any other files not already using brackets for importing.